siliconcompiler/toolscripts/rhel9/install-yosys-slang.sh

```
@@ -19,7 +19,8 @@ cd yosys-slang
 git checkout $(python3 ${src_path}/_tools.py --tool yosys-slang --field git-commit)
 git submodule update --init --recursive
 
-make -j${NPROC:-$(nproc)}
+cmake -B build .
+make -C build -j${NPROC:-$(nproc)}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)